src/js/server/schemaHandler.js

```diff
@@ -0,0 +1,53 @@
+/*
+
+    An extension of `gpii.express.handler` that adds JSON Schema headers to the outgoing response.
+
+    See this component's documentation for more details:
+
+    https://github.com/the-t-in-rtf/gpii-json-schema/blob/GPII-1336/docs/handler.md
+
+ */
+"use strict";
+var fluid = require("infusion");
+var gpii  = fluid.registerNamespace("gpii");
+
+require("gpii-express");
+
+fluid.registerNamespace("gpii.schema.handler");
+
+// Send the appropriate headers and then let the underlying grade's `sendResponse` function take over.
+gpii.schema.handler.sendResponse = function (that, response, statusCode, body) {
+    if (!that.options.schemaKey || !that.options.schemaUrl) {
+        fluid.log("Your gpii.schema.handler is not configured correctly and cannot set the appropriate headers.");
+    }
+    else if (response.headersSent) {
+        fluid.log("Can't set headers, they have already been sent.");
+    }
+    else {
+        response.type("application/" + that.options.schemaKey + "+json; profile=\"" + that.options.schemaUrl + "\"");
```

Comment:
Looking at the [relevant sections of RFC 1341](https://www.w3.org/Protocols/rfc1341/4_Content-Type.html), we see

```
Content-Type := type "/" subtype *[";" parameter] 

type :=          "application"     / "audio" 
          / "image"           / "message" 
          / "multipart"  / "text" 
          / "video"           / x-token 

x-token := <The two characters "X-" followed, with no 
           intervening white space, by any token> 

subtype := token 

parameter := attribute "=" value 

attribute := token 

value := token / quoted-string 

token := 1*<any CHAR except SPACE, CTLs, or tspecials> 

tspecials :=  "(" / ")" / "<" / ">" / "@"  ; Must be in 
           /  "," / ";" / ":" / "\" / <">  ; quoted-string, 
           /  "/" / "[" / "]" / "?" / "."  ; to use within 
```

As we commonly use periods in our ids/filenames, we are already breaking the convention.  I am looking at libraries for [quote encoding](http://www.ietf.org/rfc/rfc2616.txt) the string now.